honeycomb-core/src/twomap.rs

```diff
@@ -1255,7 +1327,41 @@ impl<const N_MARKS: usize> TwoMap<N_MARKS> {
     }
 }
 
-// --- 3-MAP
+/// Computes the total size of a given [TwoMap].
+///
+/// # Arguments
+///
+/// - `map: &TwoMap<N_MARKS>` -- Map to compute the size of.
+///
+/// ## Generics
+///
+/// - `const N_MARKS: usize` -- Number of marks used by the map structure.
+///
+/// # Return / Panic
+///
+/// Return the approximate size of the structure **in bytes**.
+///
+/// # Example
+///
+/// ```text
+///
+/// ```
+///
+pub fn map_size<const N_MARKS: usize>(map: &TwoMap<N_MARKS>) -> usize {
```

Comment:
Is it a function instead of a method to handle other kind of maps in the future?

Reply:
I initially thought this would be a utility function, but a detailed breakdown of memory usage will require reading into the private fields; I probably will turn it into a method when writing memory-related benchmarks